AndelaOSP/wire
# #154079729 - Implement search logic in search component

#### What does this PR do?
Implements Search Logic
#### Description of Task to be completed?
As an Admin, WHEN I visit the Incident search page THEN I should be able to search for Incidents.
#### Where should the reviewer start?
View file changes on PR.
#### How should this be manually tested?
- Ensure you're running in development mode by setting NODE_ENV=development in .env
- Open the dashboard and click on the search input on the navbar.
- Search for an incident, you should see relevant results as you type.
#### What are the relevant pivotal tracker stories?
#154079729
#### Screenshots (if appropriate)
<img width="1150" alt="screen shot 2018-02-27 at 2 35 01 pm" src="https://user-images.githubusercontent.com/25130454/36726673-ad3f8eea-1bcb-11e8-8c1f-eca37f7e996d.png">
<img width="1121" alt="screen shot 2018-02-27 at 2 35 19 pm" src="https://user-images.githubusercontent.com/25130454/36726683-b4019c3c-1bcb-11e8-936c-8320dc74ef67.png">
